sdk/include/opentelemetry/sdk/trace/exporter.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "opentelemetry/nostd/shared_ptr.h"
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace trace
+{
+/**
+ * ExportResult is returned as result of exporting a span batch.
+ */
+enum class ExportResult
+{
+  // Batch was successfully exported.
+  rSuccess = 0,
+  // Exporting failed. The caller must not retry exporting the same batch; the
+  // batch must be dropped.
+  rFailedNotRetryable
```

Comment:
There are [three values](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-tracing.md#span-exporter), need to add rFailedRetryable?

Reply:
I think the [spec will change very soon](https://github.com/open-telemetry/opentelemetry-specification/pull/511) to only have two values here.

Reply:
I see, thanks!